src/Routes/Register/Register.js

```diff
@@ -73,7 +72,9 @@ const Register = () => {
                                 <RegisterWithRhsm intl={intl} />
                             ) : values['how-are-systems-managed'] === 'rhs' ?
                                 <SubscribetoSatellite intl={intl} />
-                                : <EnablingInsightsOnRhui intl={intl} />
+                                : values['how-are-systems-managed'] === 'rhui' ?
+                                    <EnablingInsightsOnRhui intl={intl} />
+                                    : 'Please indicate how your systems are managed.'
```

Comment:
Is this something that we need to intl?

Reply:
OH SNAP yeah good call i forgot to do that in the rust of the work 🤗